main.go

```diff
@@ -0,0 +1,22 @@
+package main
+
+import (
+	"fmt"
+	database "go1f/pkg/db"
+	"go1f/pkg/server"
+)
+
+func main() {
+
+	dbFile := "scheduler.db"
+
+	err := database.Init(dbFile)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
В твоем случае программа с ошибкой продолжит работу, а так нельзя.
```suggestion
		log.Fatal(err)
```